lib/widgets/story_view.dart

```diff
@@ -90,7 +89,6 @@ class StoryItem {
                 ) ??
                 TextStyle(
                   color: contrast > 1.8 ? Colors.white : Colors.black,
-                  fontSize: fontSize,
```

Comment:
Then the default should be maintained. `18`.

Reply:
Yeah. Makes sense.
Fixed in [commit](https://github.com/Akora-IngDKB/story_view/commit/19ea8f6bdab1d1cbc5eab1f8a8d5cd55728aa5a4)

Reply:
Great merging.